index.html

```diff
@@ -768,12 +772,16 @@ <h2 id="cose-header-param-cwt-claims">COSE Header Parameters and CWT Claims</h2>
                 <code>exp</code>, and <code>cnf</code>.
             </p>
             <p>
-                When the <code>iat</code> and/or <code>exp</code> CWT claims are present, they
+                When the <code>iat</code> (Issued At) and/or
+		<code>exp</code> (Expiration Time) CWT claims are present, they
                 represent the issuance and expiration time of the signature, respectively. Note
                 that these are different from the <code>validFrom</code> and
                 <code>validUntil</code> properties defined in
                 <a data-cite="VC-DATA-MODEL-2.0#validity-period">Validity Period</a>, which
                 represent the validity of the data that is being secured.
+		Use of the <code>nbf</code> (Not Before) claim is NOT RECOMMENDED,
+		as it makes little sense to attempt to future-date or post-date
+		a signature.
```

Comment:
`future-date or post-date` are redundant. It's not clear whether the intent was only to discuss dates in the future, or dates in past *or* future.
```suggestion
		as it makes little sense to attempt to assign a future date to
		a signature.
```